### DISCUSSION TITLE
Major security hole (vulnerability) in Hubs

### DISCUSSION THREAD

**Discussion by Aaron-E-Walsh** [mozilla/hubs#4542] [General Questions & Discussion] [answered]
It appears that all Hubs rooms and installations are at risk due to a security vulnerability, and I therefore do not want to disclose it publicly through these discussion groups.

What is the proper way to report a serious security hole (vulnerability in Hubs) of this nature so that the vulnerability can be patched as quickly as possible?

With thanks,
Aaron

**Comment by brianpeiris** [comment, marked as answer]
Hi Aaron. You can email us at hubs@mozilla.com, or file a secure ticket on bugzilla (check the checkboxes under the "Security" section): https://bugzilla.mozilla.org/enter_bug.cgi?product=Websites you can CC me bpeiris@mozilla.com on the ticket.

**Comment by Aaron-E-Walsh** [comment]
Thank you, Brian. I just sent the email and cc'ed you as you have suggested.

Regards,
Aaron
